sharktank/sharktank/layers/paged_attention.py

```diff
@@ -904,6 +904,15 @@ def paged_attention(
                 page_ids=seq_block_ids,
             )
 
+        # Ensure all tensors are in the correct attention dtype after kv cache read
+        # Only apply to unquantized tensors - quantized tensors will be handled in attention()
+        if not isinstance(q, (QuantizedTensor, PlanarQuantizedTensor)):
```

Comment:
```suggestion
        if not isinstance(q, QuantizedTensor):
```
Don't need to include subclasses